components/Smiley.tsx

```diff
@@ -19,12 +19,31 @@ const EMOJI_MAP: EmojiMap = {
 }
 
 const Smiley = ({ percentage }: Props) => {
+  const min = 0
+  const max = 10
```

Comment:
Are you sure this doesn't conflict with the percentage/is it possible to generate the min and max values using the percentage value we already have?